### PR TITLE
Get the Chainlink oracle working again

### PR DESCRIPTION
**Pull request by michaelfig**
This was the minimal set of things needed to make the Chainlink end-to-end work again.

I didn't break up these changes into separate PRs because they didn't result in a working system until the very end.

Also, these changes rely indirectly (via published Docker images) on https://github.com/smartcontractkit/external-adapters-js/pull/1299 and https://github.com/smartcontractkit/external-initiator/pull/160 .

### CODE REVIEW

File: chainlink-agoric/README.md [hunk 1]
```diff
@@ -27,17 +27,19 @@ This can either be a public chain or a local chain **BUT NOT the simulated chain
...
 
 ## 3. Setup Chainlink node, external adapter, and external initiator
 
 _Note: Make sure you have cd-ed into this directory_
 
-Once you are running the Agoric local-chain, simply run:
+Once the above Agoric local-chain has started, simply run:
 
 ```bash
 ./setup
```
Comment (dckc) [inline chainlink-agoric/README.md:45]: When I ran `setup` it churned for a while and then said:

```
Error response from daemon: Container 55c7149d08ddbbc0e0a2b10d7f02e8194a49de19a52846df5bf1caeff778ad09 is restarting, wait until the container is running
```

Comment (dckc) [reply chainlink-agoric/README.md:45]: oh... I think you made changes since I cloned...

**Review by dckc — COMMENTED** [review]
snapshotting review comments, since I'm losing my marbles a bit...

File: priceAuthority/factory.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // @ts-check
 import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
```
Comment (dckc) [inline priceAuthority/factory.js:3]:
```suggestion
import { Far } from '@agoric/far';
```

File: priceAuthority/src/registry.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // @ts-check
 
 import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
```
Comment (dckc) [inline priceAuthority/src/registry.js:4]:
```suggestion
import { Far } from '@agoric/far';
```

File: chainlink-agoric/README.md (hunk 1 above)
Comment (dckc) [reply chainlink-agoric/README.md:45]: The docker stuff seems to depend on a `host.docker.internal` mechanism that's not available on my linux box.

cf. [What is linux equivalent of "host\.docker\.internal" \- Stack Overflow](https://stackoverflow.com/questions/48546124/what-is-linux-equivalent-of-host-docker-internal/67158212#67158212)

This seemed to get me a little further:

```diff
diff --git a/chainlink-agoric/etc/network-config.json b/chainlink-agoric/etc/network-config.json
index 5264552..a551f40 100644
--- a/chainlink-agoric/etc/network-config.json
+++ b/chainlink-agoric/etc/network-config.json
@@ -1,7 +1,7 @@
 {
   "chainName": "agoric",
-  "gci": "http://host.docker.internal:26657/genesis",
+  "gci": "http://172.17.0.1:26657/genesis",
   "rpcAddrs": [
-    "host.docker.internal:26657"
+    "172.17.0.1:26657"
   ]
 }

```

**Review by dckc — COMMENTED** [review]
I haven't managed to reproduce your results. I could go ahead and approve the PR anyway, but I'm not sure that helps. Does it? What's the goal here?

**Review by dckc — APPROVED** [review]
It seems to work, mostly.

File: chainlink-agoric/README.md (hunk 1 above)
Comment (michaelfig) [reply chainlink-agoric/README.md:45]: I made an issue.

**Pull request merged**